crates/holochain_zome_types/src/lib.rs

```diff
@@ -175,21 +175,32 @@ macro_rules! secure_primitive {
 
         impl Eq for $t {}
 
+        #[cfg(not(feature = "subtle-encoding"))]
         /// The only meaningful debug information for a cryptograhpic secret is the literal bytes.
         /// Also, encodings like base64 are not constant time so debugging could open some weird
         /// side channel issue trying to be 'human friendly'.
         /// It seems better to never try to encode secrets.
         ///
+        /// Note that using this crate with feature "subtle-encoding", a hex
+        /// representation will be used.
+        ///
         /// @todo maybe we want something like **HIDDEN** by default and putting the actual bytes
```

Comment:
I think this PR takes care of the @todo here

Reply:
@steveeJ i don't think so, subtle encoding just makes the encoding constant time, it doesn't hide any information